### DISCUSSION TITLE
Should we split the book into multiple markdown files?

### DISCUSSION THREAD

**Discussion by g-kouv** [llsoftsec/llsoftsecbook#85] [Ideas]
With more sections being added, it seems like it will be more difficult to navigate the book as a single file when editing it - should we consider a hierarchy of files? What would we need in order to make that work?

A couple of things this could be useful for:

* running tools on specific chapters/sections (e.g. search, search and replace, spell checking, and so on)
* seeing the structure of the book without having to look at the TOC in the generated pdf/webpage

I'm already starting to miss the former, and although it's still possible to work around, I can see that it might become a pain in the future: my spell checker is already picking up spelling issues in sections other than the one I'm working on (which while they need fixing, they need fixing separately), it's getting annoying to search for a word within the section that a reviewer specifically asked to replace, and so on. WDYT?

**Comment by kbeyls** [comment]
I was expecting all along that at some point it would start becoming easier to have multiple source files, rather than 1 giant source file.
I have no strong preference one way or another, apart probably for not splitting source files into a granuality smaller than 1 source file per chapter.
